### DISCUSSION TITLE
what can i do if i have accent in my username?

### DISCUSSION THREAD

**Discussion by zolcsika71** [community/community#43411] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

![copilot error](https://user-images.githubusercontent.com/20337502/210860643-300e8765-9cef-4d19-b9fc-80e3d7c83e6b.jpg)

**Comment by Akash1134** [comment, marked as answer]
Hi @zolcsika71 , It seems you are facing a similar issue as mentioned in this discussion. Please check out the [solution](https://github.com/community/community/discussions/17250#discussioncomment-2819383) and let us know if that helps. Thanks!

**Comment by zolcsika71** [comment]
Hi!

It helped.

thx,

Z.

Akash ***@***.***> ezt írta (időpont: 2023. jan. 6., P,
14:20):
